[2025-06-07T23:44:32.237+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: update_daily_data_dag.exploitation_delta_tables_task scheduled__2025-06-06T23:30:00+00:00 [queued]>
[2025-06-07T23:44:32.245+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: update_daily_data_dag.exploitation_delta_tables_task scheduled__2025-06-06T23:30:00+00:00 [queued]>
[2025-06-07T23:44:32.246+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-06-07T23:44:32.287+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): exploitation_delta_tables_task> on 2025-06-06 23:30:00+00:00
[2025-06-07T23:44:32.293+0000] {standard_task_runner.py:60} INFO - Started process 2852 to run task
[2025-06-07T23:44:32.296+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'update_daily_data_dag', 'exploitation_delta_tables_task', 'scheduled__2025-06-06T23:30:00+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/scheduled_task.py', '--cfg-path', '/tmp/tmpm20nulct']
[2025-06-07T23:44:32.298+0000] {standard_task_runner.py:88} INFO - Job 34: Subtask exploitation_delta_tables_task
[2025-06-07T23:44:32.322+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-06-07T23:44:32.352+0000] {task_command.py:423} INFO - Running <TaskInstance: update_daily_data_dag.exploitation_delta_tables_task scheduled__2025-06-06T23:30:00+00:00 [running]> on host 9eafd80829be
[2025-06-07T23:44:32.455+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='update_daily_data_dag' AIRFLOW_CTX_TASK_ID='exploitation_delta_tables_task' AIRFLOW_CTX_EXECUTION_DATE='2025-06-06T23:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-06T23:30:00+00:00'
[2025-06-07T23:44:37.398+0000] {python.py:201} INFO - Done. Returned value was: False
[2025-06-07T23:44:37.457+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=update_daily_data_dag, task_id=exploitation_delta_tables_task, execution_date=20250606T233000, start_date=20250607T234432, end_date=20250607T234437
[2025-06-07T23:44:37.518+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-07T23:44:37.532+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
